kafka-agent/src/main/java/io/strimzi/kafka/agent/KafkaAgent.java

```diff
@@ -208,9 +231,17 @@ private void touch(File file) throws IOException {
     public static void premain(String agentArgs) {
         int index = agentArgs.indexOf(':');
         if (index == -1) {
-            LOGGER.error("Unable to parse arguments {}", agentArgs);
-            System.exit(1);
+            LOGGER.debug("Running in KRaft mode");
```

Comment:
what do you think about adding a comment on what are the parameters passed to the agent or how it's called in the two different ways. I know it's in the kafka_run script but would be helpful instead of jumping to the script file having some info here.